### ISSUE TITLE
Typo in REARPORT_BUTTONS template code

### ISSUE THREAD

**Issue by mzbroch** [netbox-community/netbox#9794]
### NetBox version

v3.3-beta

### Python version

3.8

### Steps to Reproduce

1. Analyse code in `dcim/tables/template_code.py` -> `REARPORT_BUTTONS`

### Expected Behavior

b_termination_type points to `circuits.circuittermination` and not to `circuits.circuitterminations` (s)

### Observed Behavior

Actual code:
```
                <li><a class="dropdown-item" href="{% url 'dcim:cable_add' %}?a_terminations_type=dcim.rearport&a_terminations={{ record.pk }}&b_terminations_type=circuits.circuitterminations&return_url={% url 'dcim:device_rearports' pk=object.pk %}">Circuit Termination</a></li>
```

**Comment by mzbroch** [comment]
Addressed in #9796

**Comment by ryanmerolle** [comment]
Thanks @mzbroch for submitting the issue and a PR for this simple typo!
